skore/src/skore/cross_validate.py

```diff
@@ -218,17 +218,43 @@ def cross_validate(*args, project: Optional[Project] = None, **kwargs) -> dict:
 
     Examples
     --------
-    >>> def prepare_cv():
-    ...     from sklearn import datasets, linear_model
-    ...     diabetes = datasets.load_diabetes()
-    ...     X = diabetes.data[:150]
-    ...     y = diabetes.target[:150]
-    ...     lasso = linear_model.Lasso()
-    ...     return lasso, X, y
-
-    >>> project = skore.load("project.skore")  # doctest: +SKIP
-    >>> lasso, X, y = prepare_cv()  # doctest: +SKIP
-    >>> cross_validate(lasso, X, y, cv=3, project=project)  # doctest: +SKIP
+    Classification task (without saving in a skore project):
+
+    >>> from sklearn import datasets, svm
+    >>> import skore
+    >>> X, y = datasets.load_iris(return_X_y=True)
+    >>> clf = svm.SVC(kernel='linear', random_state=0)
+    >>> cv_results = skore.cross_validate(clf, X, y, cv=5)
```

Comment:
ts more explicit to define the project when calling `cross_validate`: `cv_results = skore.cross_validate(clf, X, y, cv=5, project=project)`.

Reply:
I made several examples, the first one does not save anything, the second one does

Reply:
The examples seem short enough? It's 150 samples?

Reply:
Sorry, i read too fast and didn't see the 

> (without saving in a skore project)

Reply:
Please can you measure the execution time of these doctests? Furthermore, they failed.